tests/test_operation.py

```diff
@@ -87,7 +97,8 @@ class DummyOp(qml.operation.Operator):
             num_wires = 1
 
         with pytest.raises(qml.wires.WireError, match="Wires must be unique"):
-            DummyOp(0.5, wires=[1, 1], do_queue=False)
+            with qml.QueuingManager.stop_recording():
```

Comment:
I wonder why `do_queue=False` was used here at all.  It's testing something completely separate. We can probably just get rid of the keyword argument entirely. Shouldn't change the content of test.

Reply:
You are right. It's not necessary. I removed it 0b784cf1b1bd4110717c5a45410c6c7b9baec454